cryptography/src/sha256/benches/hash_message.rs

```diff
@@ -0,0 +1,24 @@
+use commonware_cryptography::{Hasher, Sha256};
+use criterion::{criterion_group, Criterion};
+use rand::{thread_rng, Rng};
+
+fn benchmark_hash_message(c: &mut Criterion) {
+    for message_length in [100, 1000, 10000].into_iter() {
+        let mut msg = vec![0u8; message_length];
+        thread_rng().fill(msg.as_mut_slice());
```

Comment:
Please use `seed_from_u64` to create a shared sampler: https://github.com/commonwarexyz/monorepo/blob/5924bebc95371c8cf9fb46aa384f61e6c0413d66/storage/src/mmr/benches/append_additional.rs#L12-L13